index.js

```diff
@@ -11,11 +12,11 @@ module.exports = function Rivet(url, options) {
     const _fragments = temporary_processFragments(fragments)
     const _dependencies = processDependencies(dependencies)
     const _query = processVariables(dependencies, variables, query)
+    const _fullQuery = `${_query}\n${[..._fragments, ..._dependencies].join(
+      '\n'
+    )}`
 
-    return client.request(
-      `${_query}\n${[..._fragments, ..._dependencies].join('\n')}`,
-      variables
-    )
+    return client.request(_fullQuery, variables)
```

Comment:
This appears to be the meat of this PR, which LGTM.

Everything else appears to be formatting.